ui/templates/includes/learning_resource.html

```diff
@@ -26,7 +26,7 @@
       </div>
       <div class="tile-content">
         <h2>
-          <a href="{% url 'export' repo.slug result.object.id %}" class="cd-btn" data-learningresource-id="{{ result.object.id }}">
+          <a href="#" class="cd-btn" data-learningresource-id="{{ result.object.id }}">
```

Comment:
not sure it's a good practice, but for debugging reasons (at least) it would be a good idea to have the learning resource id somewhere, so what about having the `href="#<lr_id>"`?